.github/workflows/validate-package-version.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: sf org create scratch --no-namespace --definition-file config/project-scratch-def.json --alias scratch-org --set-default --duration-days 1
 
       - name: Check package installation
-        run: sf force package install --package ${{ inputs.packageId }} --wait 10
+        run: sf package install --skip-handlers --package ${{ inputs.packageId }} --wait 10
```

Comment:
suggestion/fix: split everything changing `.github`  directory and `.gitignore` file to a specific PR, so that scope are more clear and review are focused on either adding new functionalities or updating CI workflows

Reply:
cf https://github.com/salesforce/apex-mockery/pull/67